synapse_pay_rest.gemspec

```diff
@@ -34,5 +34,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rspec', '~> 3.10'
   s.add_development_dependency 'rspec-mocks', '~> 3.10'
   s.add_development_dependency 'byebug'
-  s.add_development_dependency 'webmock', '~> 3.12'
+  s.add_development_dependency 'webmock', '~> 3.6.0'
```

Comment:
Updated webmock because of issues 
* https://github.com/bblimke/webmock/issues/786
* https://github.com/bblimke/webmock/issues/822

Reply:
this looks like we went _backwards_ from 3.12 to 3.6 - is that right? was there a regression after 3.6?

Reply:
Thanks. I mixed this one up with another change. I just set it back.